modules/empire/assets/configs/playerSpawnConfig.json

```diff
@@ -0,0 +1,7 @@
+{
+    "Empire Tiny": {
+	    "hull": "empire:empireTiny",
```

Comment:
These lines use a combination of tabs and spaces. We don't like tabs around these parts.

Please make sure that none of your files contain tabs.